libunwind/src/UnwindRegistersRestore.S

```diff
@@ -753,15 +771,10 @@ DEFINE_LIBUNWIND_FUNCTION(__libunwind_Registers_arm64_jumpto)
   // context struct, because it is allocated on the stack, and an exception
   // could clobber the de-allocated portion of the stack after csp has been
   // restored.
-  ldr    c16,     [c0, #0x1f0]
-  ldp    c0, c1,  [c0, #0x000]  // restore c0,c1
-  mov    csp,c16                // restore csp
-#ifdef __ARM_MORELLO_PURECAP_BENCHMARK_ABI
-  and    x30, x30, #~1
-  ret    x30                    // jump to pc
-#else
-  ret                           // jump to pcc
-#endif
+  ldr    c2,      [c0, #0x1f0]
```

Comment:
Why restore c2 and c3 above if they're just going to be clobbered here and restored again for real later?